pkg/utils/utils.go

```diff
@@ -38,16 +37,16 @@ func SignEFIExecutable(key crypto.Signer, cert *x509.Certificate, file []byte) (
 		return nil, err
 	}
 
-	sig, err := pkcs7.SignPKCS7(key, cert, authenticode.OIDSpcIndirectDataContent, pecoffBinary.Bytes())
+	signedFile, err := pecoffBinary.Sign(key, cert)
```

Comment:
no need to manually do weird things when we cna sign it directly and the authenticode lib does it for us!